pages/Auto-Disenchanter.md

```diff
@@ -0,0 +1,3 @@
+# Auto Disenchanter
```

Comment:
Articles may not start with a heading, please refer to this guide for more help:
https://github.com/Slimefun/Slimefun4/wiki/Expanding-the-Wiki#step-3-general-guidelines-dos-and-donts